src/popup/popup.html

```diff
@@ -0,0 +1,68 @@
+<html>
+<header>
+    <link type="text/css" rel="stylesheet" charset="UTF-8" href="popup_style.css">
+    <meta charset="UTF-8">
+</header>
+
+<body>
+    <div class="grid-container-form">
+        <div class="grid-item custom-select">Moduletype</div>
+        <div class="grid-item">
+            <select id="moduleType" name="study">
+              <option value="Kernmodul">Kernmodul</option>
+              <option value="Projektmodul">Projektmodul</option>
+              <option value="Erweiterungsmodul">Erweiterungsmodul</option>
+              <option value="Majormodul">Majormodul</option>
+              <option value="Zusatzmodul">Zusatzmodul</option>
+          </select>
+        </div>
+        <div class="grid-item">Kuerzel</div>
+        <div class="grid-item">
+            <input id="modulAcronym" type="text" value="testModul"><br>
+        </div>
+        <div class="grid-item">Durchfuehrung:</div>
+        <div class="grid-item radio-container">
+            <label class="container">H
+            <input type="radio" value="H" checked="checked" name="moduleImplementation">
+            <label class="container">F
+            <input type="radio" value="F" checked="checked" name="moduleImplementation">
+        </div>
+        <div class="grid-item">Durchfuehrungsjahr</div>
+        <div class="grid-item"> <select id="moduleYear"></select><br>
+        </div>
+        <div class="grid-item">Bewertung</div>
+        <div class="grid-item">
+            <input type="number" id="moduleMark" name="grade" step="0.5" value="5" min="0" max="6">
+            <p class="comment">0 - für keine Bewertung</pclass="comment">
+        </div>
+        <div class="grid-item">Grad</div>
+        <div class="grid-item">
+            <select id="moduleGrade">
+              <option>A</option>
```

Comment:
a mix of 2 spaces and 4 spaces indent is present in this file too.. I would suggest 2 spaces for HTML. 2 spaces would also be consistent with the other HTML files in `src/templates`